src/ui/events.js

```diff
@@ -357,13 +357,13 @@ export class MapWheelEvent extends Event {
 }
 
 /**
- * `MapBoxZoomEvent` is a class used by other classes to generate
- * mouse events of specific types such as 'boxzoomstart' or 'boxzoomend'.
+ * `MapBoxZoomEvent` is a class used to generate
+ * the events 'boxzoomstart', 'boxzoomend' and 'boxzoomcancel'.
```

Comment:
Extreme nit: oxford comma 😬 

```suggestion
 * the events 'boxzoomstart', 'boxzoomend', and 'boxzoomcancel'.
```